### DISCUSSION TITLE
How to make it accent insensitive for all of my projects?

### DISCUSSION THREAD

**Discussion by ubalklen** [streetsidesoftware/vscode-spell-checker#2703] [Q&A] [answered]
According to [this](https://github.com/streetsidesoftware/cspell/issues/1060#issuecomment-1007283031), I can change a `cspell.json` to make the extension accent insensitive.

And according to [this](https://cspell.org/configuration/imports/#importing-configuration), the configuration files are local, based on the directory hierarchy of the project.

But I want this setting to be applied user-wide, so the extension behaves in a accent-insensitive way by default.

Is this possible?

**Comment by Jason3S** [comment, marked as answer]
@ubalklen,

If you change it in `user` settings, it should be applied to all projects by default.

But, you need to set it for all languages:

![image](https://github.com/streetsidesoftware/cspell/assets/3740137/005eb92d-a1c4-4992-8097-c6f6b69f1a9e)

![image](https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/a3734a15-a7ab-46f6-9f5e-372d174245f9)

```jsonc
 "cSpell.languageSettings": [
        {
            "languageId": "*", // this is the programming language / file type.
            "locale": ["pt"], // it is necessary to set the locale
            "caseSensitive": false
        }
    ]
```
